.expeditor/update_delivery-cli_to_latest.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+############################################################################
+# What is this script?
+#
+# Chef Workstation uses Expeditor to manage the bundled version of the Delivery CLI.
+# Currently we always want to include the latest version of Delivery CLI inside
+# Workstation, so this script takes that version and uses sed to insert
+# it into the omnibus project definition. Then it commits that change
+# and opens a pull request for review and merge.
+############################################################################
+
+set -evx
+
+branch="expeditor/delivery_cli_${VERSION}"
+git checkout -b "$branch"
+
+sed -i -r "s/override :\"delivery-cli\",\s+version: \"v[^\"]+\"/override :\"delivery-cli\", version: \"v${VERSION}\"/" omnibus/config/projects/chef-workstation.rb
```

Comment:
Looks like there is currently no `v` on the version string - `"0.0.45"` not `"v0.0.45"`

Reply:
`sed -i -r "s/override\s+\"delivery-cli\",\s+version:\s+\"[^\"]+\"/override \"delivery-cli\", version: \"${VERSION}\"/" omnibus/config/projects/chef-workstation.rb`